core/tests/protractor_desktop/topicAndStoryEditorFileUploadFeatures.js

```diff
@@ -71,15 +75,140 @@ describe('Topic editor functionality', function() {
+    topicEditorPage.changeSubtopicPageContents(
+      forms.toRichText('Subtopic Contents'));
+    expect(topicEditorPage.getSubtopicThumbnailSource())
+      .not.toEqual(
+        topicEditorPage.submitSubtopicThumbnail('../data/test_svg.svg')
+          .then(function() {
+            return topicEditorPage.getSubtopicThumbnailSource();
+          }));
+    topicEditorPage.saveSubtopic();
+    topicEditorPage.saveTopic('Edited subtopic.');
+
+    topicsAndSkillsDashboardPage.get();
+    topicsAndSkillsDashboardPage.editTopic('TASEFUF_2');
+    topicEditorPage.moveToSubtopicsTab();
+    topicEditorPage.expectTitleOfSubtopicWithIndexToMatch('Modified Title', 0);
+    topicEditorPage.editSubtopicWithIndex(0);
+    topicEditorPage.expectSubtopicPageContentsToMatch('Subtopic Contents');
+  });
+
   afterEach(function() {
     general.checkForConsoleErrors([]);
     users.logout();
   });
 });
+
+describe('Chapter editor functionality', function() {
+  var topicsAndSkillsDashboardPage = null;
+  var topicEditorPage = null;
+  var storyEditorPage = null;
+  var explorationEditorPage = null;
+  var dummyExplorationIds = [];
+  var dummyExplorationInfo = [
+    'Dummy exploration', 'Algorithm', 'Learn more about oppia', 'English'];
+  var TOPIC_NAME = 'TASEFUF_3';
+  var USER_EMAIL = 'creator@chapterTest.com';
+
+  var createDummyExplorations = function(numExplorations) {
+    var ids = [];
+    for (var i = 0; i < numExplorations; i++) {
+      var info = dummyExplorationInfo.slice();
+      info[0] += i.toString();
+      workflow.createAndPublishExploration.apply(workflow, info);
+      browser.getCurrentUrl().then(function(url) {
+        var id = url.split('/')[4].replace('#', '');
+        ids.push(id);
+      });
+    }
+    return ids;
+  };
+
+  beforeAll(function() {
+    topicsAndSkillsDashboardPage =
+      new TopicsAndSkillsDashboardPage.TopicsAndSkillsDashboardPage();
+    topicEditorPage = new TopicEditorPage.TopicEditorPage();
+    storyEditorPage = new StoryEditorPage.StoryEditorPage();
+    skillEditorPage = new SkillEditorPage.SkillEditorPage();
+    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+    explorationEditorMainTab = explorationEditorPage.getMainTab();
+    users.createAndLoginAdminUser(
+      USER_EMAIL, 'creatorChapterTest');
+    dummyExplorationIds = createDummyExplorations(1);
+    users.logout();
+  });
+
+  beforeEach(function() {
+    users.login(USER_EMAIL);
+  });
+
+  it('should create a basic chapter with a thumbnail.', function() {
+    topicsAndSkillsDashboardPage.get();
+    topicsAndSkillsDashboardPage.createTopic(TOPIC_NAME, 'abbrev');
+    expect(topicEditorPage.getTopicThumbnailSource())
+      .not
+      .toEqual(
+        topicEditorPage.submitTopicThumbnail('../data/test_svg.svg')
+          .then(function() {
+            return topicEditorPage.getTopicThumbnailSource();
+          })
+      );
+    topicEditorPage.changeAbbreviatedTopicName('short name');
```

Comment:
I believe we are removing the abbreviated name field. See https://github.com/oppia/oppia/pull/9087#discussion_r409515494